cfn-resources/global-cluster-config/template.yml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: AWS SAM template for the Mongodb::Atlas::GlobalClusterConfig resource type
```

Comment:
What is the purpose / difference between this file and files in example folder?

[mongodbatlas-cloudformation-resources](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/2b6be069bb82b266a8ca4a78135ac57ea15a6a84)/[examples](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/2b6be069bb82b266a8ca4a78135ac57ea15a6a84/examples)/global-cluster-config/

Reply:
This template file is for local testing using SAML. It is same as template file available in other modules.